builds/checkin/api-proxy.yaml

```diff
@@ -0,0 +1,84 @@
+trigger: none
+pr:
+  branches:
+    include:
+      - master
+      - release/*
+      - iiot
```

Comment:
Just curious, if this PR committed to master but not yet merge to iiot branch, then a PR created from any branch other than master.  Then github will trigger checkin builds, when it try to trigger api-proxy.yaml, it will fail, right?

Reply:
correct, it will need to be ported first